### PR TITLE
Update documentation

### LINKED ISSUE #2: Update Documentation
- Update User Guides
- Update Developer Guides
- Update README and AboutMe documents

### PR DESCRIPTION
**Pull request by laurenlhy**
Update developer guide, user guide, AboutUs

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -38,22 +38,67 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
...
 ### Use cases
 
 (For all use cases below, the **System** is the `CHIM` and the **Actor** is the `user`, unless specified otherwise)
 
+#### Use case: Input a Customer
+
+**MSS**
+
+1. User adds a customer by specifying name, phone number and address.
+2. CHIM creates the new customer and shows details of the new customer.
```
Comment (daniellau88) [inline docs/DeveloperGuide.md]: Do remove the trailing whitespaces in order to pass the build checks

**Comment by daniellau88** [comment]
The AboutUs image is not working... I think you would need to rename it to `laurenlhy.png` (currently its named `laurenlhy.png.JPG`)

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -38,22 +38,67 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
...
+| `* * *`  | New user                    | Remove an existing order |  |
+| `* * *`  | New user                    | Add cheese entries |  |
+| `* * *`  | New user                    | Mark a sample order as delivered |  |
+| `* * *`  | New user                    | Search for a customer | Find the customer’s contact information |
+| `* *`    | New user                    | Save the data input | Retrieve the same information later |
+| `* *`    | User with some familiarity  | View a summary of my inventory | See if there is a need to increase production |
+| `* *`    | User                        | See the introduction message |  |
+| `*`      | User with some familiarity  | Search up orders of a specific customer | Efficiently find the order status |
+
+[More to be added]
```
Comment (daniellau88) [inline docs/DeveloperGuide.md:67]: Is this needed?

Comment (laurenlhy) [reply docs/DeveloperGuide.md:67]: Maybe they can serve as examples of low priority tasks?

Comment (aaronsms) [reply docs/DeveloperGuide.md:67]: Should be fine I guess.

**Review by daniellau88 — APPROVED** [review]
LGTM

**Comment by aaronsms** [comment]
Actually do you want to squash some of your commits, before merging. But other than that, everything seems fine.

**Comment by daniellau88** [comment]
> Actually do you want to squash some of your commits, before merging. But other than that, everything seems fine.

Yea, I think would be good to squash to about 3 commits

**Pull request merged**